contracts/Rewards.sol

```diff
@@ -36,56 +34,45 @@ contract Rewards is Context {
 
     modifier onlyGovernance() {
```

Comment:
suggestion: as this modifier behaviour is common across a number of contract, abstract into a shared contract (similar to (OpenZepplin Ownable)[https://github.com/OpenZeppelin/openzeppelin-contracts/blob/master/contracts/access/Ownable.sol] ).